app/src/main/java/org/fossasia/openevent/fragments/SpeakerFragment.java

```diff
@@ -45,20 +51,27 @@
  */
 public class SpeakerFragment extends Fragment implements SearchView.OnQueryTextListener {
 
+
+    private static final String PREF_SORT = "sortType";
+
     final private String SEARCH = "searchText";
 
+    private SharedPreferences prefsSort;
+
+    private List<Speaker> mSpeakers;
+
     private SwipeRefreshLayout swipeRefreshLayout;
 
     private RecyclerView speakersRecyclerView;
 
     private SpeakersListAdapter speakersListAdapter;
 
-    private List<Speaker> mSpeakers;
-
```

Comment:
In  android studio, go to Code ---> Rearrange code. 
It will automatically rearrange the code.